Sources/MapboxCoreNavigation/EHorizon/DistancedRoadObject.swift

```diff
@@ -13,15 +13,15 @@ public enum DistancedRoadObject {
      - parameter type: Road object type.
      - parameter distance: Distance to the point object, measured in meters.
      */
-    case point(identifier: RoadObjectIdentifier, type: RoadObjectType, distance: CLLocationDistance)
+    case point(identifier: RoadObjectIdentifier, type: RoadObject.ObjectType, distance: CLLocationDistance)
```

Comment:
I didn’t realize it earlier, but `RoadObjectIdentifier` could also be renamed to `RoadObject.Identifier`. `RoadObjectType` was the main type that would’ve been awkward, so it isn’t a huge deal. But if you’re up to it, it would be even nicer to go through all the `RoadObject*` types and nest them under `RoadObject`. If not, we can treat it as tail work: it isn’t so bad to keep a deprecated type alias around with the original, unnested name.